src/node.cc

```diff
@@ -168,6 +169,7 @@ static uv_async_t dispatch_debug_messages_async;
 static node::atomic<Isolate*> node_isolate;
 static v8::Platform* default_platform;
 
+int v8_thread_pool_size = v8_default_thread_pool_size;
```

Comment:
> Would it work to ditch v8_default_thread_pool_size, and just initialize this to DefaultPlatform::kMaxThreadPoolSize, like @ofrobots suggested?

That would depend if we wanted to change our defaults based on upstream v8 changes, or have a default value that is controlled from within core. Let's say v8 changed kMaxThreadPoolSize to MAX_INT, I am sure that we would not want to set the default based on this?

> I think you'll want to group this variable with the variables declared starting at line 134.

Thanks! Was looking for a nicer place. Will update that now.